openprescribing/frontend/management/commands/measure_definitions/lpzomnibus.json

```diff
@@ -7,7 +7,7 @@
     "An aggregation of prescribing on all low priority treatments as defined by NHS England"
   ],
   "why_it_matters": [
-    "NHS England have recently published a <a href='https://www.england.nhs.uk/wp-content/uploads/2017/07/Items-not-routinely-prescribed-in-primary-care.pdf'>number of areas which should not be routinely prescribed in primary care</a>.",
+    "NHS England have recently published a <a href='https://www.england.nhs.uk/medicines/items-which-should-not-be-routinely-prescribed/.",
```

Comment:
You've lost the closing `</a>` tag!  Without this, the page will not render properly.

Reply:
I think I lost part of the text too, sorry..